reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogFactory.java

```diff
@@ -16,18 +16,57 @@
 package reactor.netty.http.server.logging;
 
 import java.util.function.Function;
+import java.util.function.Predicate;
 
-/**
+ /**
  * An interface to declare more concisely a {@link Function} that apply an {@link AccessLog} by an
  * {@link AccessLogArgProvider}.
  * <p>
- * Can be used in {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory) accessLog} method
- * for example.
+ * Can be used in {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory) accessLog} method for example.
  *
  * @author Simon Baslé
  * @author Audrey Neveu
  * @since 1.0.3
  */
 public interface AccessLogFactory extends Function<AccessLogArgProvider, AccessLog> {
 
-}
\ No newline at end of file
+	/**
```

Comment:
there seems to be something wrong with how your IDE is configured: there's an extra space before each comment line in this block

Reply:
ah, nevermind, only the first line was problematic. the other lines are fine, but the rendering in the ui (probably with a plugin I use) displays a small artifact that looks like an extra space:
![image](https://user-images.githubusercontent.com/6986166/103680705-d99ba400-4f86-11eb-9ab5-b0f2008d5e24.png)